Doc/c-api/structures.rst

```diff
@@ -399,6 +399,35 @@ definition with the same method name.
    slot.  This is helpful because calls to PyCFunctions are optimized more
    than wrapper object calls.
 
+.. c:function:: PyObject * PyCMethod_New(PyMethodDef *ml, PyObject *self, PyObject *module, PyTypeObject *cls)
+
+   Turn *ml* into a Python :term:`callable` object.  The caller must ensure that
+   *ml* outlives the :term:`callable`.  Typically, *ml* is defined as a static
+   variable.
+
+   The *self* parameter will be passed as the *self* argument to the C function
+   in ``ml->ml_meth`` when invoked.  *self* can be ``NULL``.
+
+   The :term:`callable` object's ``__module__`` attribute can be set from the
+   given *module* argument.  *module* should be a Python string, which will be
+   used as name of the module the function is defined in.  If unavailable, it
+   can be set to :const:`None` or ``NULL``.
+
+   The *cls* parameter will be passed as the *defining_class* argument to the C
+   function.  Must be set if :c:macro:`METH_METHOD` is set on ``ml->ml_flags``.
+
+   .. seealso: :attr:`function.__module__`
```

Comment:
Please move this just below the *module* argument doc.

Reply:
The seealso not rendered at https://cpython-previews--112557.org.readthedocs.build/en/112557/c-api/structures.html#c.PyCMethod_New , maybe it must at the end of a section?

Reply:
Ah, I got it, there is a missing colon.